sdk/storage/ci.yml

```diff
@@ -53,3 +53,6 @@ extends:
       safeName: AzureResourceManagerStorage
     TestSetupSteps:
     - template: /sdk/storage/tests-install-azurite.yml
+    MatrixReplace:
+      # Use dedicated storage pool in canadacentral with higher memory capacity
+      - Pool=(.*)-general/$1-storage
```

Comment:
We can certainly try this, but in CI mode the region of the VM shouldn't matter, and the SKUs between these pools are identical.

You can also invoke a run directly from this branch instead of merging, which I think is preferable, so that we don't accidentally leave tests in that use up allocation in the storage pool.

Reply:
@weshaggard tracked down the issue - it is related to a T1 mgmt project reference that storage tests were using that was not fully signed.